_rules/element-lang-matches-default-language-off6ek.md

```diff
@@ -325,11 +327,25 @@ The `lang` [attribute value][] of this `p` element is a [grandfathered tag][gran
 </html>
 ```
 
+#### Inapplicable Example 8
+
+The [text inheriting its programmatic language][] from this `div` element it only [whitespace][].
+
+```html
+<html>
```

Comment:
I don't think SC 3.1.1 allows this TBH. There is no "except if the page is empty" or "except if the page has no default language" exception in that SC. The other rule I agree with, but I don't think we should do the same to this rule.